File: algorithms/944-delete-columns-to-make-sorted/delete-columns-to-make-sorted.go
package problem944

func minDeletionSize(strs []string) int {
	output := 0
	row := len(strs)
	col := len(strs[0])
	for j := 0; j < col; j++ {
		for i := 1; i < row; i++ {
			if strs[i-1][j] > strs[i][j] {
				output++
				break
			}
		}
	}
	return output
}

File: algorithms/944-delete-columns-to-make-sorted/delete-columns-to-make-sorted.py
from typing import List

def min__deletion_size(strs: List[str]) -> int:
    row = len(strs)
    col = len(strs[0])
    output = 0
    for j in range(0, col):
        for i in range(1, row):
            if strs[i - 1][j] > strs[i][j]:
                output += 1
                break
    return output
